### ISSUE TITLE
[Feature] Option for a custom Discord Application ID in settings

### ISSUE THREAD

**Issue by nayutune** [yukino-app/yukino#95]
**Is your feature request related to a problem? Please describe.**
People like making their profiles look fancy, i think allowing people to customize the look of their profile, without making them compile stuff is nice because people don't like placeholder images and would rather put something fancy they can showoff

**Describe the solution you'd like**
It would be nice to have an option in settings that allows users to easily replace discord developer application id with their own.

**Additional context**
![Example](https://user-images.githubusercontent.com/74027819/129038923-82129e06-5266-41f6-aedf-db76cd1d8ea3.png)

**Comment by zyrouge** [comment]
I don't know how useful it will be as it is gonna produce same results as disable the in-built rpc and using a self-created one. Replacing the application id would not change it as it applies the same settings. (The reason it does not change the image is because discord does not allow using those as the assets should be used from the application assets)

**Comment by nayutune** [comment]
Yeah i'm familiar with discord's limitations on RPC, i just thought ability for users to upload their own "large" image key for their own Developer App ID without having to recompile source code to change one line that features Discord App ID would be nice.
Just due to having a few people i know would love to make their profile look fancy when watching something, and surely those are not the only one's who would love to have this kind of easy to edit thing.
![image](https://user-images.githubusercontent.com/74027819/129081978-1d4d39f7-b202-489f-af51-4baae36aa955.png)

I know it will still return same details, button etc..
But being able to modify "Large" image and "Small" image could be nice without having to recompile the app
and obviously it's not a high priority request or anything, just sort of a convenience addition that i feel like would be nice to have in the future, because there are way more important requests that need to be added
